cyhy_report/customer/generate_report.py

```diff
@@ -3432,6 +3432,18 @@ def __generate_mustache_json(self, filename):
             t["first_detected_time_tex"] = t["first_detected"].strftime("{%H}{%M}{%S}")
             t["last_detected_date_tex"] = t["last_detected"].strftime("{%d}{%m}{%Y}")
             t["last_detected_time_tex"] = t["last_detected"].strftime("{%H}{%M}{%S}")
+            # Trim long descriptions so that they don't overflow the page
+            # (LaTeX table cells cannot span pages, as far as I know) or cause
+            # other issues. For more information, see:
+            # - https://github.com/cisagov/cyhy-reports/issues/124
+            # - https://github.com/cisagov/cyhy-reports/issues/123
+            # NOTE: I wanted to make the cutoff 4096 characters, but that barely
+            # fit on a single page when there was a single affected host.  I
+            # went with 3072 instead to save space on the page for when there
+            # are many affected hosts listed.
+            if len(t["description"]) > 3072:
```

Comment:
Can we define this magic number as a constant at the top of the file?

Reply:
Good call- please see https://github.com/cisagov/cyhy-reports/pull/125/commits/3028cda8740d990c26af31d4ff1e2baf452cf8b2.